plugins/woocommerce/src/Admin/API/AnalyticsImports.php

```diff
@@ -0,0 +1,279 @@
+<?php
+/**
+ * REST API Analytics Imports Controller
+ *
+ * Handles requests to get batch import status and trigger manual imports.
+ */
+
+declare( strict_types = 1 );
+
+namespace Automattic\WooCommerce\Admin\API;
+
+use Automattic\WooCommerce\Internal\Admin\Schedulers\OrdersScheduler;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * REST API Analytics Imports Controller.
+ *
+ * @internal
+ * @extends WC_REST_Data_Controller
+ */
+class AnalyticsImports extends \WC_REST_Data_Controller {
+	/**
+	 * Endpoint namespace.
+	 *
+	 * @var string
+	 */
+	protected $namespace = 'wc-analytics';
+
+	/**
+	 * Route base.
+	 *
+	 * @var string
+	 */
+	protected $rest_base = 'imports';
+
+	/**
+	 * Register routes.
+	 */
+	public function register_routes() {
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/status',
+			array(
+				array(
+					'methods'             => \WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_status' ),
+					'permission_callback' => array( $this, 'get_item_permissions_check' ),
+				),
+				'schema' => array( $this, 'get_status_schema' ),
+			)
+		);
+
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/trigger',
+			array(
+				array(
+					'methods'             => \WP_REST_Server::CREATABLE,
+					'callback'            => array( $this, 'trigger_import' ),
+					'permission_callback' => array( $this, 'create_item_permissions_check' ),
+				),
+				'schema' => array( $this, 'get_trigger_schema' ),
+			)
+		);
+	}
+
+	/**
+	 * Check if a given request has access to read import status.
+	 *
+	 * @param  \WP_REST_Request $request Full details about the request.
+	 * @return \WP_Error|boolean
+	 */
+	public function get_item_permissions_check( $request ) {
+		if ( ! current_user_can( 'manage_woocommerce' ) ) {
+			return new \WP_Error(
+				'woocommerce_rest_cannot_view',
+				__( 'Sorry, you cannot view analytics import status.', 'woocommerce' ),
+				array( 'status' => rest_authorization_required_code() )
+			);
+		}
+
+		return true;
+	}
+
+	/**
+	 * Check if a given request has access to trigger import.
+	 *
+	 * @param  \WP_REST_Request $request Full details about the request.
+	 * @return \WP_Error|boolean
+	 */
+	public function create_item_permissions_check( $request ) {
+		if ( ! current_user_can( 'manage_woocommerce' ) ) {
```

Comment:
This looks the same as `get_item_permissions_check`. Is there another capability to check? Otherwise, could we merge the two?

Reply:
Done in 5c0410644e1d5eac5557fc8f36ae977fd009d9d5